test-requirements.txt

```diff
@@ -144,27 +127,25 @@ sphinxcontrib-qthelp==1.0.3
     # via sphinx
 sphinxcontrib-serializinghtml==1.1.5
     # via sphinx
-tomli==2.0.1
+tomli==2.0.1 ; python_version < '3.11' or (python_version < '3.11' and implementation_name == 'cpython')
```

Comment:
isn't only `python_version < '3.11'` marker enough at here? I think ` or (python_version < '3.11' and implementation_name == 'cpython')` never affects.

Reply:
I didn't make this myself, this was a part of how uv created it. I agree that the or part is redundant, but that's an issue with uv.

Reply:
Correct, this already reported in https://github.com/astral-sh/uv/issues/4536

Reply:
👍 Should be fixed soon.

Reply:
Either way IMO this shouldn't block a merge